### PR TITLE
126 add start again confirmation

### PR DESCRIPTION
**Pull request by Ancentury**
French button "Annuler" will show when browser language set to French. However, if browser language set to English, it is still showing "Cancel" even on French page. We could have work around, but it needs third library and make it complicated. Please let me know if it's ok.

### CODE REVIEW

File: src/components/ActionButtonBar.vue [hunk 1]
```diff
@@ -62,6 +62,15 @@ import { Model } from "survey-vue";
 @Component
 export default class ActionButtonBar extends Vue {
   @Prop() survey?: Model;
+  startModal() {
+    let text =
+      this.$i18n.locale === "fr"
+        ? "Cette action effacera toutes les données non enregistrées. Êtes-vous sûr de vouloir continuer ?"
```
Comment (nschonni) [inline src/components/ActionButtonBar.vue:68]: I don't think https://github.com/canada-ca/aia-eia-js/blob/master/src/survey-enfr.json is the right spot, but it seems like all these hardcoded translation strings should be external lookups

Comment (Ancentury) [reply src/components/ActionButtonBar.vue:68]: Good point. As this is only applied to this button, not used anywhere else, I keep it in it's own file to make it simpler.

**Comment by MrDeshaies** [comment]
@nschonni howzit now with my modifications?

**Comment by nschonni** [comment]
Makes sense. Probably a separate issue to go through the code and figure out where else the `$t` lookups need to be used

**Comment by MrDeshaies** [comment]
@Ancentury OK/Cancel being in the language of the browser is fine; that's how JS alerts work.

**Review by MrDeshaies — APPROVED** [review]

**Pull request merged**